src/pages/Home/components/Map/components/Location/Location.module.scss

```
@@ -0,0 +1,112 @@
+.container {
+  position: relative;
+  top: 140px;
+  left: 700px;
```

가운데 정렬이 고정이 되어 있어서 화면을 줄이면 잘려서 fixed 형식으로 수정해야할 것 같아요.
주소 검색창과 모달창이 겹쳐서 margin을 줘야할 필요가 있을 것 같습니다. 아니면 top값을 변경해도 좋고요.
```suggestion
  position: fixed;
  top: 50%;
  left: 50%;
  transform: translate(-50%, -50%);
```

---

넵 수정했습니다! 
477f29e9c687ada18e4fafd5ef7afb8405ad61f5